etc/mock/site-defaults.cfg

```diff
@@ -268,7 +268,7 @@
 # config_opts['chroot_additional_packages'] = ''
 # config_opts['log_config_file'] = 'logging.ini'
 # config_opts['more_buildreqs']['srpm_name-version-release'] = 'dependencies'
-# config_opts['macros']['%Add_your_macro_name_here'] = "add macro value here"
+# config_opts['macros']['%build_name.tst'] = 'My own hostnamme'
```

Comment:
you should not take a generic template and make it specific. if you wanted to show something specific you should add a new one.  you could use it to define %host %packager %host_cpu %distribution among others.